lib/index.js

```diff
@@ -20,7 +20,8 @@ const DEFAULT_OPTIONS = {
   createParentPath: false,
   parseNested: false,
   useTempFiles: false,
-  tempFileDir: path.join(process.cwd(), 'tmp')
+  tempFileDir: path.join(process.cwd(), 'tmp'),
+  hashAlgorithm: 'md5'
```

Comment:
What if add a comma at the end of the last line.

Reply:
This would violate the project's configured eslint rules.
`24:23  error  Unexpected trailing comma  comma-dangle`